app/src/test/java/echo/ConsoleTest.java

```diff
@@ -0,0 +1,38 @@
+package echo;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import java.io.*;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+public class ConsoleTest {
+  Console console = new Console();
+
```

Comment:
totally ok with me if you don't want to add this to this PR bc everything else looks great but on the next PR could we add a teardown method to this suite that restores the original System.in and System.out?

Reply:
Yay thank you!!! 

yes definitely! I'll add it to the next ticket :)